GOVERNANCE.md

```diff
@@ -0,0 +1,87 @@
+# CoreDNS Governance
+
+## Principles
+
+The CoreDNS community adheres to the following principles:
+
+- Open: CoreDNS is open source. See repository guidelines, below.
```

Comment:
It's unclear to me what section below this is referring to.

Reply:
right. will rather add a link to coredns.io/community that advertize it.